tgui/packages/tgui/interfaces/MemoryPanel.js

```diff
@@ -0,0 +1,127 @@
+import { useBackend } from '../backend';
+import { Box, Button, Dimmer, Icon, Section, Stack } from '../components';
+import { multiline } from 'common/string';
+import { Window } from '../layouts';
+
+const STORY_VALUE_SHIT = 0;
+const STORY_VALUE_NONE = 1;
+const STORY_VALUE_MEH = 2;
+const STORY_VALUE_OKAY = 3;
+const STORY_VALUE_AMAZING = 4;
+// STORY_VALUE_LEGENDARY = 5; (not actually used)
+
+const MemoryQuality = (props, context) => {
+  const { act } = useBackend(context);
+  const {
+    quality,
+  } = props;
+
+  if (quality === STORY_VALUE_SHIT) {
+    return (
+      <Button
+        icon="poop"
+        color="brown"
+        tooltip={multiline`
+          This memory is not interesting at all! It does not make for
+          good art and is unlikely to pass to future generations.
+        `} />
+    );
+  } else if (quality === STORY_VALUE_NONE) {
```

Comment:
there is no need for else, since you return anyway

Reply:
you forgot about this thread